src/Uno.UI/UI/Xaml/Controls/Popup/Popup.wasm.cs

```diff
@@ -1,19 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Uno.Extensions;
-using System.Linq;
-using System.Drawing;
+﻿using Uno.Extensions;
 using Uno.Disposables;
-using Windows.UI.Xaml.Media;
 using Uno.Logging;
+using Windows.UI.Xaml.Controls.Primitives;
 
 namespace Windows.UI.Xaml.Controls
 {
 	public partial class Popup
 	{
 		private readonly SerialDisposable _closePopup = new SerialDisposable();
 
+		internal FlyoutPlacementMode AnchorPlacement { get; set; } = FlyoutPlacementMode.Bottom;
```

Comment:
Popup is placed with Horizontal and vertivaloffset, not anchors, no? The flyout is using Placement. Would we have a way to be more in line with this? (and not add this internal property?)
_You position the Popup by setting the HorizontalOffset and VerticalOffset properties. The Popup is offset relative to its immediate parent container. A Popup is not modal, so input to the screen behind it is not blocked._
[ref](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.popup)